src/components/elements/lazy-symbol.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import icons from 'components/elements/symbols'
+
+type TLazySymbolProps = {
+    // name: keyof typeof icons
+    name: string
+    width?: string
+    height?: string
+    alt?: string
```

Comment:
Let's have the `alt` as `required`

Reply:
Can we make sure, this is only for svg ?

Reply:
Sure, I'll add a check for SVGs